### PR TITLE
feat: Roads redesign v2

### PR DESCRIPTION
**Pull request by Kinerius**
## What does this PR change?
Cliff models altered and pushed in to the edge of the last parcel
Ocean water grid lowered to accommodate new cliff position
Terrain Data Generators changed to use correct cliff models

Please also make sure to use all Road assets from this branch, as main right now has names mixed up

Happy testing :)

...

## How to test the changes?

1. Launch the explorer
2. ...

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Comment by olavra** [comment]
Tested and looking good for me

**Comment by meshr00m** [comment]
Ready to test and merge, added description on top!

**Review by anicalbano — APPROVED** [review]
Looks like the assets names are still the same as main:

image

**Review by anicalbano — APPROVED** [review]
🟢 Approved on the QA side

- There are no crashes
- Roads are working as expected

https://github.com/decentraland/unity-explorer/assets/139157886/fa1f3ab4-1205-4be5-8b2f-2995f071cf64



https://github.com/decentraland/unity-explorer/assets/139157886/739c9fd5-2fdc-4b37-831e-dbd4247aca8e

File: Explorer/Assets/DCL/PluginSystem/Global/LandscapePlugin.cs [hunk 1]
```diff
@@ -80,7 +80,9 @@ public void InjectToWorld(ref ArchSystemsWorldBuilder<Arch.Core.World> builder,
 
             LandscapeDebugSystem.InjectToWorld(ref builder, debugContainerBuilder, floor, realmPartitionSettings.Value, landscapeData.Value);
             LandscapeTerrainCullingSystem.InjectToWorld(ref builder, landscapeData.Value, terrainGenerator);
-            LandscapeMiscCullingSystem.InjectToWorld(ref builder, landscapeData.Value, terrainGenerator);
+
+            // Disabled temporally
+            //LandscapeMiscCullingSystem.InjectToWorld(ref builder, landscapeData.Value, terrainGenerator);
```
Comment (m3taphysics) [inline Explorer/Assets/DCL/PluginSystem/Global/LandscapePlugin.cs:85]: Is this right?

**Review by m3taphysics — CHANGES_REQUESTED** [review]

File: Explorer/Assets/DCL/PluginSystem/Global/LandscapePlugin.cs (hunk 1 above)
Comment (Kinerius) [reply Explorer/Assets/DCL/PluginSystem/Global/LandscapePlugin.cs:85]: It is

**Pull request merged**